ui/app/index.html

```diff
@@ -20,6 +20,7 @@
             var app = Elm.Main.init({
                 flags: {
                     production: true,
+                    startWeekAtMonday: JSON.parse(localStorage.getItem('startWeekAtMonday')),
```

Comment:
Can we store `startWeekOnSunday` instead, to make sure Monday is the default?

Reply:
Since the variable is now called "firstDayOfWeek" everywhere, I also refactored the flag to be called "firstDayOfWeek".
I hope this is fine as well.